CONTRIBUTING.md

```diff
@@ -1,146 +1,107 @@
 
-- [Code of Conduct](#code-of-conduct)
-- [Question or Problem?](#got-an-api-product-question-or-problem)
-- [Issues and Bugs](#found-an-issue)
-- [Feature Requests](#want-a-feature)
-- [Documentation fixes](#want-a-doc-fix)
-- [Submission Guidelines](#submission-guidelines)
-- [Coding Rules](#coding-rules)
+To set up the repository, run:
 
-## Code of Conduct
-
-Please be kind to and considerate of other developers, as we all have the same goal: make `telnyx-node` as good as
-it can be.
-
-## Got an API/Product Question or Problem?
-
-If you have questions about how to use `telnyx-node`, please see our
-[docs](./README.md), and if you don't find the answer there, please contact
-[Telnyx Support](https://telnyx.com/contact-us) with any issues you have.
+```sh
+$ yarn
+$ yarn build
+```
 
-## Found an Issue?
+This will install all the required dependencies and build output files to `dist/`.
 
-If you find a bug in the source code or a mistake in the documentation, you can
-help us by submitting [an issue][issue-link]. If the file in which the error
-exists is autogenerated, the change will need to be made by us, but
-submitting an issue will help us track it and keep you up-to-date. If the file
-isn't generated, you can help us out even more by submitting a Pull Request with
-a fix.
+## Modifying/Adding code
 
-**Please see the [Submission Guidelines](#submit) below.**
+Most of the SDK is generated code. Modifications to code will be persisted between generations, but may
+result in merge conflicts between manual patches and changes from the generator. The generator will never
+modify the contents of the `src/lib/` and `examples/` directories.
 
-## Want a Feature?
+## Adding and running examples
 
-You can request a new feature by submitting an issue to our
-[GitHub Repository][github]. If you would like to implement a new feature then
-consider what kind of change it is:
+All files in the `examples/` directory are not modified by the generator and can be freely edited or added to.
 
-- **Major Changes** that you wish to contribute to the project should be
-  discussed first with `telnyx-node` contributors in an issue or pull request so
-  that we can develop a proper solution and better coordinate our efforts,
-  prevent duplication of work, and help you to craft the change so that it is
-  successfully accepted into the project.
-- **Small Changes** can be crafted and submitted to the
-  [GitHub Repository][github] as a Pull Request.
+```ts
+// add an example to examples/<your-example>.ts
 
-## Want a Doc Fix?
+#!/usr/bin/env -S npm run tsn -T
+…
+```
 
-If you want to help improve the docs in the helper library, it's a good idea to
-let others know what you're working on to minimize duplication of effort. Create
-a new issue (or comment on a related existing one) to let others know what
-you're working on.
+```sh
+$ chmod +x examples/<your-example>.ts
+# run the example against your api
+$ yarn tsn -T examples/<your-example>.ts
```

Comment:
could you add an example?

Reply:
the examples directory is entirely untouched by codegen, so you can add an example file via custom code if you wish. this would be by pushing something to https://github.com/team-telnyx/telnyx-node/tree/next